packages/SystemsThinkingWorkbench-Core.package/STWPlayfield.class/instance/addSimulationButtons.st

```diff
@@ -0,0 +1,15 @@
+initialization
+addSimulationButtons
+	
+	self playButton: STWButton new.
+	self playButton contents: 'Start'.
+	self playButton clickHandler: [ self toggleSimulation. ].
+	self addMorph: self playButton.
+	self playButton position: self position.
+	
+	self stopButton: STWButton new.
+	self stopButton contents: 'Reset'.
+	self addMorph: self stopButton.
+	self stopButton clickHandler: [self resetSimulation. ].
+	self stopButton position: self position + ((self playButton extent x + 10)@ 0).
```

Comment:
the customer wants 1 button with changing text after clicked, wich toggles... why do we have a start AND a stop button